src/main/java/com/carservice/manager/controller/VehicleController.java

```diff
@@ -25,35 +29,53 @@ public String test() {
 
     @GetMapping("/cars")
     public String carsView(Model model) {
-        model.addAttribute("cars", vehicleService.getVehicleModels());
+        model.addAttribute("cars", vehicleService.getVehicleModels("main"));
```

Comment:
```suggestion
        model.addAttribute("cars", vehicleService.getVehicles());
```
It is better to have the controller independent from the file/database repository 😉